base/sparse/higherorderfns.jl

```diff
@@ -93,7 +93,8 @@ end
 # (3) broadcast[!] entry points
 broadcast(f::Tf, A::SparseVector) where {Tf} = _noshapecheck_map(f, A)
 broadcast(f::Tf, A::SparseMatrixCSC) where {Tf} = _noshapecheck_map(f, A)
-function broadcast!(f::Tf, C::SparseVecOrMat) where Tf
+
+function broadcast!(f::Tf, C::SparseVecOrMat, ::Void) where Tf
```

Comment:
Use `::Void`for the broadcast style  here and in the next `broadcast!` method since we're specializing on the destination only, as per the rules at the end of https://github.com/JuliaLang/julia/issues/24914#issuecomment-349722131.